client/templates/adminMenu/adminMenu.html

```diff
@@ -1,4 +1,4 @@
-<div layout="row">
+<div layout="row" layout-align="center center">
```

Comment:
Yeah, 0ea03dc fixed it but the comment still stays up because I tagged the line with the opening div instead of the now deleted extra closing div. Ya div.